src/doc/book/getting-started.md

```diff
@@ -11,7 +11,7 @@ an Internet connection to run the commands in this section, as we’ll be
 downloading Rust from the Internet.
 
 We’ll be showing off a number of commands using a terminal, and those lines all
-start with `$`. We don't need to type in the `$`s, they are there to indicate
+start with `$`. You don't need to type in the `$`s, they are there to indicate
```

Comment:
This is the "author's we", and we pretty much use it everywhere. I can see why this might be a bit more clear. It's fine.